csharp/download_deps.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
My build script lives on! It has so many children.

Reply:
saved our bacon